tidb-cloud/tidb-cloud-security-faq.md

```diff
@@ -0,0 +1,81 @@
+
+As a customer, you own your data and have the right to choose which TiDB Cloud cluster can process, store, and host the data. TiDB Cloud will not access or use your data without your permission. 
+
+## Who has control over customer data?
+
+As a customer, you have full control of your data:
+
+- You determine the level of security of your data based on your industry's data security and privacy policies. TiDB Cloud offers encryption features to protect your data either in transit or stored at different security levels.
+
+- You manage access to your data, as well as access to the TiDB Cloud databases and tools through users, roles, and credentials that you control.
+
+## Where is customer data stored?
+
+In TiDB Cloud, you have the flexibility to choose how and where you want to run your TiDB database. When you do, you have access to your TiDB database tools, storage locations, regional control planes, APIs, Chat2Query, and other database services. If you want to run your database globally, you can choose from various regions on AWS or GCP.
+
+As a customer, you can decide in which AWS or GCP region your data will be stored by creating your TiDB cluster in the target region that meets your geographic needs. For example, if you are in the USA and you want to store all your data only in the USA, you can choose to deploy a TiDB cluster exclusively in the AWS Oregon (us-west-2) region.
+
+You can also replicate and back up your data in multiple AWS or GCP regions. PingCAP will not move or replicate your data outside of your chosen AWS or GCP region without your permission.
+
+## What are the roles in securing customer data?
+
+When you assess the security and compliance of roles and responsibilities, it is crucial to differentiate between the security role of TiDB Cloud and the security role of your own in using TiDB Cloud. 
+
+TiDB Cloud is mainly responsible for implementing and maintaining the sustainable, safe, stable, and compliant operation of TiDB Cloud.
+
+As a customer, you are your database owner, and you take the administrative and operational role of your database. You are primarily responsible for ensuring the security and compliance of your TiDB cluster within the TiDB Cloud. And your role in TiDB Cloud also involves implementing and managing any security features and access control policies of your own TiDB cluster.
+
+## Who can access customer data in TiDB Cloud?
+
+The security of customer data is of the highest priority to TiDB Cloud. Regardless of which TiDB Cloud region you have selected, TiDB Cloud implements rigorous technical, contractual, and organizational measures to safeguard your data confidentiality, integrity, and availability.
+
+PingCAP has passed the standard certification and verification of SOC2 Type2, GDPR, PCI-DSS, and HIPAA, and invites third-party auditors to audit the certified standards every year. For details, see [PingCAP Trust Center](https://www.pingcap.com/trust-compliance-center).
```

Comment:
```suggestion
PingCAP has passed the standard certification and verification of SOC2 Type II, GDPR, PCI-DSS, and HIPAA, and invites third-party auditors to audit the certified standards every year. For details, see [PingCAP Trust Center](https://www.pingcap.com/trust-compliance-center).
```

Reply:
be consistent with the usage of SOC 2 Type II at https://www.pingcap.com/trust-compliance-center/